library.js

```diff
@@ -103,14 +103,15 @@ DiscordAuth.getStrategy = function (strategies, callback) {
 
   meta.settings.get('sso-discord-alt', function (err, settings) {
     if (err) return callback(err)
-    if (!settings.id || !settings.secret) {
+
+    options.clientID = settings.id || process.env.SSO_DISCORD_CLIENT_ID || ''
+    options.clientSecret = settings.secret || process.env.SSO_DISCORD_CLIENT_SECRET || ''
```

Comment:
Does this go against the standard for nodebb plugins?

Reply:
I'm not sure.
You're still can override them by the form in ACP.